includes/class-wc-countries.php

```diff
@@ -927,6 +927,7 @@ public function get_country_locale() {
 						),
 						'postcode' => array(
 							'required' => false,
+							'hidden'   => true,
```

Comment:
Just want to pause and check in on this (including the related change up on [line 518](https://github.com/woocommerce/woocommerce/pull/30875/files#diff-8597c2856232dcdf013cdc5f43618309b16b58b806cca38558f9ab5676d00033R518)). 

I did of course read the [comment in the linked issue](https://github.com/woocommerce/woocommerce/issues/30375#issuecomment-937858938) mentioning, _"We don't use postal code here,"_ and I certainly don't claim to know better, but I note that the [Correos de Chile](https://www.correos.cl/) website has a postcode finder tool and that the [UPU datasheet for Chile](https://www.upu.int/UPU/media/upu/PostalEntitiesFiles/addressingUnit/chlEn.pdf) features postcodes in the address format.

From various other (perhaps less authoritative) links it sounds like commune-level (if not street-level) postcodes may be useful and/or that postcodes are more commonly used in the capital.

Are we sure removing the postcode is better than retaining it as a non-required field?

Reply:
@barryhughes sounds right.. checked it further and got the same insights. might not be used regularely by the people but officially seems supported. therefore better to keep it as of now. also other repos work with it - so seems right to keep it... https://github.com/lgaticaq/codigo-postal

updated your suggestions :)